src/inmanta/user_setup.py

```diff
@@ -0,0 +1,163 @@
+"""
+    Copyright 2023 Inmanta
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+        http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+
+    Contact: code@inmanta.com
+"""
+import asyncio
+import secrets
+import socket
+
+import asyncpg
+import click
+
+import nacl.pwhash
+from inmanta import config, data
+from inmanta.data import CORE_SCHEMA_NAME, PACKAGE_WITH_UPDATE_FILES, schema
+from inmanta.server import config as server_config
+
+
+def generate_signing_config() -> str:
+    hostname = socket.gethostname()
+    return f"""[auth_jwt_default]
+algorithm=HS256
+sign=true
+client_types=agent,compiler,api
+key={secrets.token_urlsafe(32)}
```

Comment:
I am wondering whether it's a good idea to hard code the key length here. The advantages of hard coding the value is that our tests can verify that everything works. On the other hand this means that we have to remember to increment this value over the years to make sure that the key length is still safe. The alternative would be to use the default value of the library. According to the documentation, they will increment the default over time to keep a safe default. I am not sure in which direction we should go here. Any opinions?

Reply:
I think using the default value is the easiest and safest option